### ISSUE TITLE
Echelle diagrams are mirrored

### ISSUE THREAD

**Issue by danhey** [KeplerGO/lightkurve#594]
#### Problem description
G'day all. I'm really enjoying the new seismology (and echelle!) code in lightkurve. However, it looks like the echelle diagram produced by plot_echelle is mirrored about the x-axis. You can see this with a simple test as below;

#### Example

```python
import lightkurve as lk
from scipy.signal import find_peaks

lc = lk.search_lightcurvefile('KIC 11615890', cadence='long').download_all().stitch().remove_nans()
pg = lc.to_periodogram()
ax = pg.plot()
plt.show()

cut = pg.frequency.value > 10
p = find_peaks(pg.power.value[cut], height=5, distance=100)[0]
ax.scatter(pg.frequency.value[cut][p], pg.power.value[cut][p], c='r', marker='v')
plt.show()

seis = pg.flatten().to_seismology()
seis.estimate_numax()
seis.estimate_deltanu()

ax = seis.plot_echelle()
ax.scatter(pg.frequency.value[cut][p] % seis.deltanu.value, pg.frequency.value[cut][p], c='r', marker='x')
```
#### Expected behavior
The peaks pulled directly from the light curve modulo the returned delta nu should align with the ridges. This is not the case.

Here are the relevant plots from above:
![Screen Shot 2019-09-14 at 12 36 39 pm](https://user-images.githubusercontent.com/38233719/64902546-56e53a80-d6ec-11e9-8f32-28fce5f62a6e.png)
![Screen Shot 2019-09-14 at 12 38 17 pm](https://user-images.githubusercontent.com/38233719/64902556-96138b80-d6ec-11e9-9991-b23cb4a43b34.png)

Shameless plug, but the ridges are properly aligned in my `echelle` code (though no where near as pretty!):
![Screen Shot 2019-09-14 at 12 39 52 pm](https://user-images.githubusercontent.com/38233719/64902560-c3f8d000-d6ec-11e9-9616-3ad16e81d067.png)

#### Environment

-  platform (e.g. Linux, OSX, Windows): OSX
-  lightkurve version (e.g. 1.0b6): 1.1.1
-  installation method (e.g. pip, conda, source): source

**Comment by barentsen** [comment]
Thanks @danhey!

@christinahedges Can you look into this when you're back from your trip?

**Comment by ojhall94** [comment]
Thanks for raising this @danhey! Super weird! I'll give it a look into if I find the time later :)

**Comment by gully** [comment]
I compared @danhey 's `echelle` implementation with the lightkurve version.  As a spot check, I made Dan's `echelle()` function a drop-in replacement function and fed it right into the `imshow`, and got matching results:

![image](https://user-images.githubusercontent.com/860227/66157461-b0000a00-e5d8-11e9-9f28-75ad33c8f26b.png)


This experiment tells me that the pre-processing and deltanu estimation steps in lightcurve are all fine.  The problem must arise somewhere between the lines with:

`ff = freq[int(fmin/fs)-x0:int(fmax/fs)-x0]`
and 
`extent = (x_f.min().value, x_f.max().value, y_f[0].min().value, y_f.max().value)`

I will work my way backwards to isolate the issue.  My hunch is the reshape, since Dan's and the lightkurve reshapes look significantly different.  But we will see through careful scrutiny.
